### PR TITLE
Task pojo

### PR DESCRIPTION
**Pull request by ihormedvid**

### CODE REVIEW

File: src/main/java/com/javatrainee/POJOs/Streams.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package javatrainee;
```
Comment (yurii-polishchuk) [inline src/main/java/com/javatrainee/POJOs/Streams.java]: package

File: src/main/java/com/javatrainee/POJOs/Video.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package javatrainee;
```
Comment (yurii-polishchuk) [inline src/main/java/com/javatrainee/POJOs/Video.java]: rename package POJOs to `domain` 

https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

File: src/main/java/com/javatrainee/POJOs/Video.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
```
Comment (yurii-polishchuk) [inline src/main/java/com/javatrainee/POJOs/Video.java]: @Data

File: src/main/java/com/javatrainee/POJOs/Video.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Video {
+    private UUID id;
+    private String title;
+    private String description;
+    private LocalDateTime publishedAt;
+    private List stream;
```
Comment (yurii-polishchuk) [inline src/main/java/com/javatrainee/POJOs/Video.java]: List of what?

File: src/main/java/com/javatrainee/POJOs/Video.java (hunk 2 above)
Comment (yurii-polishchuk) [reply src/main/java/com/javatrainee/POJOs/Video.java]: please be more attentive and read the description of a task!
```
Java POJO:
package com.globallogic.domain
```

https://github.com/yurii-polishchuk/project-trainee-java/issues/17

File: .gitignore [hunk 5]
```diff
@@ -21,3 +21,8 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply .gitignore]: remove from this PR

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply pom.xml]: fix

File: src/main/java/com/javatrainee/App.java [hunk 7]
```diff
@@ -0,0 +1,14 @@
+package com.javatrainee;
+
+/**
+ * Hello world!
+ *
+ */
+public class App
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/main/java/com/javatrainee/App.java]: Fixed

[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/main/java/com/javatrainee/App.java]: fixed

File: src/main/java/com/javatrainee/POJOs/Channel.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/main/java/com/javatrainee/POJOs/Channel.java]: fixed

File: src/main/java/com/javatrainee/POJOs/Playlist.java [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Playlist {
+    private UUID id;
+    private String title;
+    private String description;
+    private List videos;
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/main/java/com/javatrainee/POJOs/Playlist.java]: fixed

File: src/main/java/com/javatrainee/POJOs/Streams.java [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+
+@Getter
```
[the remark this answers is not in the document]
Comment (ihormedvid) [reply src/main/java/com/javatrainee/POJOs/Streams.java]: fixed

File: src/main/java/com/javatrainee/POJOs/Streams.java (hunk 1 above)
Comment (ihormedvid) [reply src/main/java/com/javatrainee/POJOs/Streams.java]: fixed

File: src/main/java/com/javatrainee/POJOs/Video.java (hunk 2 above)
Comment (ihormedvid) [reply src/main/java/com/javatrainee/POJOs/Video.java]: fixed

File: src/main/java/com/javatrainee/POJOs/Video.java (hunk 4 above)
Comment (ihormedvid) [reply src/main/java/com/javatrainee/POJOs/Video.java]: fixed

**Comment by yurii-polishchuk** [comment]
@ihormedvid This branch has conflicts that must be resolved

**Pull request merged**